src/components/panels/ResetPanel.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <HubPanelWrap :title="headertext">
+    <b-list-group>
+      <b-list-group-item :style="styleObject" class="list-group-item-borderbottom">
+        <span class="pull-left">
+          <b>Reward Timer Reset in:</b>
+        </span>
+        <TimeBadge :starttime="this.time2" :endtime="this.time1"/>
```

Comment:
"time2" and "time1" are not very intuitive, especially when `:starttime` uses the larger "index" compared to `:endtime`.